js/tabs.js

```diff
@@ -465,12 +469,28 @@ Tabs.prototype.saveEntry_ = function(tab, entry, opt_callback) {
   this.save(tab, opt_callback);
 };
 
-Tabs.prototype.onDocChanged_ = function(e, session) {
+/**
+ * @param {Event} event The DocChange event object.
+ * @param {Object} eventData The data passed with the DocChange event. This
+ *   object contains where the event came from, i.e textarea or codemirror, which
+ *   helps the syncUnifiedSession function determine which of the editors is the
+ *   source of truth. It also contains the current session of the editor so this
+ *   function can check to make sure Tabs state is still accurate.
+ * The event handler which is called when a DocChange event is caught, this
```

Comment:
Nit: function description at top of jsdoc

Reply:
Good pick up! Thanks